Sources/KeyboardSupport/KeyboardToolbar.swift

```diff
@@ -33,6 +39,9 @@ open class KeyboardToolbar: UIToolbar, KeyboardAccessory {
     // MARK: - KeyboardAccessory
     
     open weak var keyboardAccessoryDelegate: KeyboardAccessoryDelegate?
+    open var nextButton: UIBarButtonItem?
```

Comment:
I had not envisioned KeyboardToolbar to have these properties. I have KeyboardToolbars on my project that just have a done button. I think a user of the framework could create their own subclass or extensions to solve their issues. What are people's opinion of having these buttons included and the methods in the extensions for storeButton and setNextAndBackButtonsHidden?

Reply:
I added the properties to the keyboardToolbar class because the KeyboardToolbarDelegate included methods for "next" and "back"

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/179be6970fc071bb9ce43417b249bc48c2795ee8

Didn't create a second Navigating toolbar concrete class, but implemented other recommendations.